### PR TITLE
Add TopK activation function

### PR DESCRIPTION
**Pull request by TomDLT**

### CODE REVIEW

File: neuron_explainer/models/autoencoder.py [hunk 1]
```diff
@@ -93,8 +93,14 @@ def from_state_dict(
         n_latents, d_model = state_dict["encoder.weight"].shape
         autoencoder = cls(n_latents, d_model)
         autoencoder.load_state_dict(state_dict, strict=strict)
+        autoencoder.activation = state_dict.get("activation", nn.ReLU())
```
Comment (WuTheFWasThat) [inline neuron_explainer/models/autoencoder.py:96]: is strict=strict on the line above going to be an issue?

maybe 
```
autoencoder.activation = state_dict.pop("activation", nn.ReLU())
autoencoder.load_state_dict(state_dict, strict=strict)
```

Comment (WuTheFWasThat) [reply neuron_explainer/models/autoencoder.py:96]: also do you think we should just have strings + a mapping to modules?  worry a bit about weird things with pickling modules

**Comment by WuTheFWasThat** [comment]
mirror of https://github.com/openai/sparse_autoencoder/pull/2

File: neuron_explainer/models/autoencoder.py (hunk 1 above)
Comment (WuTheFWasThat) [reply neuron_explainer/models/autoencoder.py:96]: huh, i guess this is how it is done in our actual codebase also

Comment (TomDLT) [reply neuron_explainer/models/autoencoder.py]: I removed the pickling

**Review by WuTheFWasThat — APPROVED** [review]
thanks!

**Pull request merged**